# Used for distributing LED values

# Standard ROS message header
Header header
# An array of LedValue messages which represent the current LED values
LedValue[] values
